plugin-build/plugin/src/main/kotlin/com/mikepenz/aboutlibraries/plugin/mapping/SpdxLicense.kt

```diff
@@ -208,9 +208,19 @@ enum class SpdxLicense(
     Leptonica("Leptonica License", "Leptonica"),
     LGPL_2_0_only("GNU Library General Public License v2 only", "LGPL-2.0-only"),
     LGPL_2_0_or_later("GNU Library General Public License v2 or later", "LGPL-2.0-or-later"),
-    LGPL_2_1_only("GNU Lesser General Public License v2.1 only", "LGPL-2.1-only"),
+    LGPL_2_1_only("GNU Lesser General Public License v2.1 only", "LGPL-2.1-only",
+        customMatcher = { name, _ ->
+            name.equals("GNU Lesser General Public License v2.1 only", true)
+                    || name.equals("GNU Lesser General Public License (LGPL), Version 2.1", true)
+        }
+    ),
     LGPL_2_1_or_later("GNU Lesser General Public License v2.1 or later", "LGPL-2.1-or-later"),
-    LGPL_3_0_only("GNU Lesser General Public License v3.0 only", "LGPL-3.0-only"),
+    LGPL_3_0_only("GNU Lesser General Public License v3.0 only", "LGPL-3.0-only",
+        customMatcher = { name, _ ->
+            name.equals("GNU Lesser General Public License v3.0 only", true)
```

Comment:
this is not needed. exact name matching is done even with a custom matcher. so only the `name.equals("GNU General Lesser Public License (LGPL) version 3.0", true)` is needed.